core/src/main/java/org/apache/carbondata/core/datamap/DataMapStoreManager.java

```diff
@@ -318,32 +319,32 @@ public TableDataMap getDefaultDataMap(CarbonTable table) {
    * Get the datamap for reading data.
    */
   public TableDataMap getDataMap(CarbonTable table, DataMapSchema dataMapSchema) {
-    String tableUniqueName =
-        table.getAbsoluteTableIdentifier().getCarbonTableIdentifier().getTableUniqueName();
```

Comment:
suggest to remove the `getTableUniqueName` in table identifier thus removing using it in all places

Reply:
Many other places want to use uniqueTableName. There would be too many changes this way